### PR TITLE
fixups for LW Linux UUID

### PR DESCRIPTION
**Pull request by markhdavid**

### CODE REVIEW

**Comment by dbmcclain** [comment]
noted. additional commits since then

**Review by dbmcclain — COMMENTED** [review]
looks correct

**Comment by markhdavid** [comment]
Ok, I'm going to merge, since David says it looks good, and 'checks have failed' is unfortunately normal now, and unfortunately, I don't have a Linux Lispworks license to test with. A pretty low-risk thing to put in at this point.

**Comment by markhdavid** [comment]
OK Done. Will delete this branch soon. (Please do not use this branch meanwhile without checking with me.)

**Review by dbmcclain — COMMENTED** [review]
not correct for Windows

File: src/useful-macros/uuid.lisp [hunk 1]
```diff
@@ -234,7 +234,7 @@ INTERNAL-TIME-UINITS-PER-SECOND which gives the ticks per count for the current
                                                           (aref ends 0)))
                                       :radix 16)))))
 
-         #+:LISPWORKS
+         #+(and :WINDOWS :LISPWORKS)
```
Comment (dbmcclain) [inline src/useful-macros/uuid.lisp:237]: Yes, correct

**Review by dbmcclain — COMMENTED** [review]
yes, correct for windows after all

File: src/useful-macros/uuid.lisp [hunk 2]
```diff
@@ -246,6 +246,17 @@ INTERNAL-TIME-UINITS-PER-SECOND which gives the ticks per count for the current
                                                 (aref starts 0)
                                                 (aref ends 0)))
                             :radix 16)))
+
```
Comment (dbmcclain) [inline src/useful-macros/uuid.lisp:249]: yes correct

**Review by dbmcclain — COMMENTED** [review]
looks good

**Pull request merged**